### ISSUE TITLE
Filters in Drawer doesn't work

### ISSUE THREAD

**Issue by DenisYakimchuk** [Shopify/dawn#1351]
**When click on a filter option - drawer is just closing and nothing else happening**

**When click on a filter option - products are filtering and chosen filter should be added to active filters **

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 3.0.0
- Chrome Version 91.0.4472.114
- macOS Version 11.3.1

**Additional context/screenshots**
The issue is actual only for the drawer, mobile and desktop view, any browser and OS.

**Comment by gregjotau** [comment]
Is this reltaed to: https://github.com/Shopify/dawn/issues/1311?

**Comment by DenisYakimchuk** [comment]
@gregjotau No, thats actually not the same issue. The bug I've reported is related to filtering functionality itself. Click on any option doesn't do anything. You can't filter products if you're using a drawer. Try to collapse it on a desktop and use some filters except price filter.
